akka-projection-grpc/src/main/protobuf/akka/projection/grpc/event_producer.proto

```diff
@@ -75,7 +75,14 @@ message FilterReq {
 
 // Replay events for given entities.
 message ReplayReq {
-  repeated PersistenceIdSeqNr persistence_id_offset = 1;
+  repeated ReplayPersistenceId replay_persistence_ids = 1;
```

Comment:
this should be a wire compatible change, but I will verify with a sample to be sure

Reply:
Are you sure about that, the original was a string + uint64, now it is a message (length delimited) + uint64 so old version messages can't be parsed out of data serialized from this.

Reply:
you are right, it's not compatible (I tried with the sample)

Would have to keep the old persistence_id_offset as it was, and add the new replay_persistence_ids, then include the data in both, and read from both. Not sure if it's worth the trouble? What do you think?

Reply:
I changed to the wire compatible solution in https://github.com/akka/akka-projection/pull/1133/commits/e10c881134261b8032785d310795d39ddb17bedf

Not too messy.